test/vue.test.ts

```diff
@@ -19,9 +19,9 @@ test("vue for loop", async ({ actual, expected }) => {
   expect(actual).toMatchInlineSnapshot(`
     {
       "branches": "0/0 (100%)",
-      "functions": "0/0 (100%)",
-      "lines": "0/2 (0%)",
-      "statements": "0/2 (0%)",
+      "functions": "0/1 (0%)",
```

Comment:
`v-for` is transformed to `_renderList(..., () => { ... })` and this callback is now mapped correctly.